packages/pyolite-kernel/py/pyolite/pyolite/kernel.py

```diff
@@ -15,3 +17,30 @@ def comm_info(self, target_name=""):
                 comms[comm_id] = dict(target_name=comm.target_name)
 
         return comms
+
+    def do_complete(self, code, cursor_pos):
```

Comment:
Maybe we can now rename it to `complete`, just like we have `run`?

Reply:
Doing.